wire-ios/Wire-iOS/Sources/UserInterface/UserProfile/Devices/UserClientCell.swift

```diff
@@ -23,142 +23,174 @@ import WireDataModel
 
 final class UserClientCell: SeparatorCollectionViewCell {
 
-    typealias IconColors = SemanticColors.Icon
-    typealias ViewColors = SemanticColors.View
-    typealias LabelColors = SemanticColors.Label
+    enum EdgeInsetConstants {
+        static let small: CGFloat = 2.0
+        static let medium: CGFloat = 4.0
+        static let normal: CGFloat = 8.0
+        static let `default`: CGFloat = 16.0
+    }
 
-    private let titleLabel = DynamicFontLabel(fontSpec: .bodyTwoSemibold,
-                                              color: LabelColors.textDefault)
-    private let subtitleLabel = DynamicFontLabel(fontSpec: .mediumRegularFont,
-                                                 color: LabelColors.textCellSubtitle)
+    typealias LabelColors = SemanticColors.Label
+    typealias IconColors = SemanticColors.Icon
 
-    private let deviceTypeIconView = UIImageView()
+    // MARK: - Properties
+    let nameLabel = DynamicFontLabel(style: .headline,
+                                     color: LabelColors.textDefault)
+    let labelLabel = DynamicFontLabel(style: .subheadline,
+                                      color: LabelColors.textDefault)
+    let mlsThumbprintLabel = DynamicFontLabel(style: .caption1,
+                                        color: LabelColors.textCellSubtitle)
+    let proteusIdLabel = DynamicFontLabel(style: .caption1,
+                                            color: LabelColors.textCellSubtitle)
```

Comment:
👍 Thank you for getting rid of old initializers and using the new ones! On that note: Did you check how it would look like on a larger font size and if we need to make any changes there?

Reply:
Yup. Looks fine